mod-source-record-storage-server/src/main/java/org/folio/services/util/AdditionalFieldsUtil.java

```diff
@@ -314,6 +314,14 @@ public static void fillHrIdFieldInMarcRecord(Pair<Record, JsonObject> recordInst
     }
   }
 
+  public static void fill035FieldInMarcRecord(Record record, String incoming001) {
```

Comment:
@Aliaksandr-Fedasiuk  I think there should be better to name this method smth like "fill035FieldInMarcRecordIfNotExists". What do you think?

Reply:
thank you, @VRohach! that’s a good point